src/extensions/default/DebugCommands/main.js

```diff
@@ -48,6 +51,13 @@ define(function (require, exports, module) {
     
     var KeyboardPrefs = JSON.parse(require("text!keyboard.json"));
     
+    // default preferences file name
+    var DEFAULT_PREFERENCES_FILENAME = "defaultPreferences.json",
+        SUPPORTED_PREFERENCE_TYPES   = ["number", "boolean", "string", "array", "object"];
+
+    var reComputeDefaultPrefs        = true,
```

Comment:
nit: `recompute` should be all-lowercase (not camelCase). See https://en.wiktionary.org/wiki/recompute